packages/venia-ui/lib/components/Header/accountTrigger.css

```diff
@@ -15,7 +15,7 @@
     composes: root from '../clickable.css';
 
     height: 3rem;
-    min-width: 3rem;
+    min-width: 6rem;
```

Comment:
There's a discussion to be had here regarding focus outlines.

Buttons (like this one) will invoke the browser default outline when they are focused. That is the "default" blue outline we see in Venia:

![Screen Shot 2020-07-14 at 9 00 58 AM](https://user-images.githubusercontent.com/13182778/87448767-e0da8a00-c5c1-11ea-8c3f-3e98e2b9eacd.png)

It's important to note that Venia does not control how this outline looks. There are places where we intentionally set `outline: none` on focus - usually when we want to render a custom `box-shadow`.

UX desired there to be some additional padding here between the focus and the element's contents. Setting `min-width` here accomplishes this

![Screen Shot 2020-07-14 at 8 53 39 AM](https://user-images.githubusercontent.com/13182778/87448966-2ac37000-c5c2-11ea-871a-c960c509d0e6.png)

but only in the case where the text is static and the button's size is known ahead of time.
When the user is signed in, this button includes their user name - which is dynamic - and the button's size can't be known ahead of time. When the button exceeds this `6rem` minimum width, there is once again no space between the contents and the outline:

![Screen Shot 2020-07-14 at 9 00 58 AM](https://user-images.githubusercontent.com/13182778/87449196-783fdd00-c5c2-11ea-9ae5-1a0bd983fb58.png)

There is a way to add some space there, but it requires Venia taking over the `outline`, not leaving it to the browser:

![Screen Shot 2020-07-14 at 11 03 28 AM](https://user-images.githubusercontent.com/13182778/87449284-99a0c900-c5c2-11ea-9d8a-05a1636ba3cb.png)

But there are a couple problems with this too:

![Screen Shot 2020-07-14 at 11 03 17 AM](https://user-images.githubusercontent.com/13182778/87449297-9f96aa00-c5c2-11ea-837e-88be80131c3b.png)

1. It doesn't match the browser default, so this balloons into a much bigger change than just this specific button if we want consistency across the app
2. There's no way to separate horizontal and vertical offset - the offset is applied uniformly around the entire element (there's also no way to set rounded corners on an `outline`)

As such, I recommend creating a new ticket to bring focus handling in to be fully controlled by Venia and to use `drop-shadow` to render it instead of `outline` for more control.

Reply:
Thank you @supernova-at for investigating this.  This it certainly more a fussy aesthetics issue than anything.  Consistency is important to re-enforce confidence, but should be balanced against effort, as well as being practical about the value of that effort.  All-in-all this all looks very good and meets UX approval.
Thnx.

Reply:
@supernova-at Good summary of the issue.

Our top priority is to have _some_ treatment, whether it's the browser's default `outline` or something custom. We have a solid custom style elsewhere that could be applied here—we'd just need to examine the case and discuss it. For now though, as with accordions, the current style is fine while we figure it out, even though it fails to meet the criteria for a `focused` style.

---

**Treatment for `focused` should**

- stand out anywhere on the page
- be applicable to a wide variety of elements
- be independent from and able to supersede other states, such as `hovered` or `selected`

On a page full of nothing but grays and blue borders, a blue outline doesn't meet those criteria. But it'll do for now. 👍

Reply:
Okay, I created a separate ticket for this.